app/components/channel_actions/manage_members_label/manage_members_label.tsx

```diff
@@ -62,8 +62,9 @@ const ManageMembersLabel = ({canRemoveUser, channelId, manageOption, testID, use
     const serverUrl = useServerUrl();
 
     const handleRemoveUser = useCallback(async () => {
-        removeMemberFromChannel(serverUrl, channelId, userId);
-        fetchChannelStats(serverUrl, channelId, false);
+        removeMemberFromChannel(serverUrl, channelId, userId).then(
+            () => fetchChannelStats(serverUrl, channelId, false),
```

Comment:
this should only be called when the `removeMemberFromChannel` does not return an error.